source/content/guides/multisite/06-search-replace.md

```diff
@@ -97,9 +101,31 @@ For subdomain Multisites, environments to be replaced are defined and paired in
 
 1. Commit the  `sites.yml` file in the `private/sites.yml` in the site’s Git repository.
 
-### Subdirectory WPMS
+### Subdomain to Subdirectory Conversion
+Sites can also be configured to use subdomain Multisite on the live environment, and subdirectory in all other instances.
 
-No additional configuration is needed if you already completed the steps in [Enable Search and Replace](/guides/multisite/search-replace/#enable-search-and-replace). Search and replace will match the behavior of the platform’s search and replace for non-WPMS sites.
+This can be configured by making `SUBDOMAIN_INSTALL` conditional in `wp-config.php`,
+
+    ```php:title=wp-config.php
+    if ( ! empty( $_ENV['PANTHEON_ENVIRONMENT'] ) && $_ENV['PANTHEON_ENVIRONMENT'] ==  'live' ) {
+      define( 'SUBDOMAIN_INSTALL', true );
+    }
+    ```
+and setting `convert_to_subdirectory: true` in `sites.yml`.
+
+    ```yaml:title=private/sites.yml
+    ---
+    api_version: 1
+    convert_to_subdirectory: true
```

Comment:
If the sites.yml is only going to have a single config line, could that config go into pantheon.yml instead? So they don't have to create & maintain a wholey separate config file for a single option?

Reply:
@pwtyler Should I consider these conversations resolved and merge when the checks have completed?